### DISCUSSION TITLE
ktLint version

### DISCUSSION THREAD

**Discussion by janseeger** [JLLeitschuh/ktlint-gradle#704] [General]
Hi, I just saw the comment in the [plugin libs.version.toml](https://github.com/JLLeitschuh/ktlint-gradle/blob/main/plugin/gradle/libs.versions.toml#L3) and wanted to ask if there is a place to read up on this?

The ktLint version that is shipped with the plugin is quite old and contains bugs, which supposedly are already fixed in [later ktLint versions](https://github.com/pinterest/ktlint/issues/1809#issuecomment-1466862008).

**Comment by wakingrufus** [comment]
The version i the toml is what we compile against, but the version that is applied by default is 0.47.1. This is still a bit old, but you can set it higher using the extension configuration. Up to 0.50.0 is supported. We will increase the default next time we do a major version release.
